search/tests/validator/validator.spec.js

```diff
@@ -4,7 +4,9 @@ const {
   loadOpenApiYaml,
   getSchema,
   getSchemaCollection,
-  validateSchema
+  validateSchema,
+  createStacValidator,
+  validateStac
```

Comment:
This file should probably be split into two test files, one for each file being tested, to fit with the convention in the rest of the application. I.e. a `schemaValidator.spec.js` file and a `stacValidator.spec.js` file.